src/Pork2Sausage.h

```diff
@@ -96,9 +96,9 @@ struct CmdParam {
 
 
 
-void launchProgram(const CmdParam & cmdParam);
+void launchProgram(const CmdParam & cmdParamValue);
 
-int getCmdsFromConf(const TCHAR *confPath, CmdParam *cmdParam, int maxNbCmd);
+int getCmdsFromConf(const TCHAR *confPathValue, CmdParam *cmdParamValue, int maxNbCmd);
```

Comment:
@chcg 
I was talking about changing the argument variable name - it's not necessary at all.

Reply:
Just reverted this changes locally and still get:

1>------ Build started: Project: pork2sausage, Configuration: Release ARM64 ------
1>Pork2Sausage.cpp
1>Process.cpp
1>c:\v141\pork2sausage\src\pork2sausage.cpp(195): error C2220: warning treated as error - no 'object' file generated
1>c:\v141\pork2sausage\src\pork2sausage.cpp(195): warning C4459: declaration of 'confPath' hides global declaration
1>c:\v141\pork2sausage\src\pork2sausage.cpp(39): note: see declaration of 'confPath'
1>c:\v141\pork2sausage\src\pork2sausage.cpp(195): warning C4459: declaration of 'cmdParam' hides global declaration
1>c:\v141\pork2sausage\src\pork2sausage.cpp(41): note: see declaration of 'cmdParam'
1>c:\v141\pork2sausage\src\pork2sausage.cpp(254): warning C4459: declaration of 'cmdParam' hides global declaration
1>c:\v141\pork2sausage\src\pork2sausage.cpp(41): note: see declaration of 'cmdParam'
1>Done building project "pork2sausage.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========

Reply:
Thank you for the explain.